### PR TITLE
Update OPA store to remove all v1 relics

### PR DESCRIPTION
**Pull request by tylercloke**
### :nut_and_bolt: Description: What code changed, and why?

Also deleted much of the readme that was inaccurate or superceded by our API and external docs

### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/authz-service/README.md [hunk 1]
```diff
@@ -11,39 +11,7 @@ The TOC below is auto-generated by the VSCode "Markdown TOC" plugin; do not edit
...
-- [API](#api)
-  - [func GetVersion](#func-getversion)
-  - [func IsAuthorized](#func-isauthorized)
-  - [type IsAuthorizedReq](#type-isauthorizedreq)
-  - [type IsAuthorizedResp](#type-isauthorizedresp)
-  - [type VersionInfo](#type-versioninfo)
-- [Other Resources](#other-resources)
-
-<!-- /TOC -->
+<!-- TOC depthFrom:2 depthTo:3 -->autoauto- [AuthZ Approach](#authz-approach)auto    - [Terminology](#terminology)auto    - [Introspection (How to Query Permissions)](#introspection-how-to-query-permissions)auto- [Introducing Projects](#introducing-projects)auto- [Data Layer](#data-layer)auto- [Other Resources](#other-resources)autoauto<!-- /TOC -->
```
Comment (bcmdarroch) [inline components/authz-service/README.md:14]: this is gonna mess up the table of contents formatting. supposedly the Markdown TOC plugin will auto-generate it but i haven't gotten that plugin to work.. @msorens might have guidance

**Comment by bcmdarroch** [comment]
Oh +1 to @srenatus 's comment, we should drop those rego files too

File: components/authz-service/README.md (hunk 1 above)
Comment (msorens) [reply components/authz-service/README.md:14]: Given the amount of deleted material, I am fine to just delete the TOC in this file now. 😁

File: components/authz-service/engine/opa/opa.go [hunk 2]
```diff
@@ -23,20 +23,16 @@ import (
...
+	preparedEvalProjects rego.PreparedEvalQuery
 }
 
 // this needs to match the hardcoded OPA policy document we've put in place
 const (
-	authzQuery              = "data.authz.authorized"
-	filteredPairsQuery      = "data.authz.introspection.authorized_pair[_]"
 	authzProjectsV2Query    = "data.authz_v2.authorized_project[project]"
 	filteredPairsV2Query    = "data.authz_v2.introspection.authorized_pair[_]"
 	filteredProjectsV2Query = "data.authz_v2.introspection.authorized_project"
```
Comment (msorens) [inline components/authz-service/engine/opa/opa.go:38]: More V2 removal at a minimum in the variable names (though the paths would be nice too):
```suggestion
	authzProjectsQuery    = "data.authz_v2.authorized_project[project]"
	filteredPairsQuery    = "data.authz_v2.introspection.authorized_pair[_]"
	filteredProjectsQuery = "data.authz_v2.introspection.authorized_project"
```

File: components/authz-service/engine/opa/opa.go [hunk 3]
```diff
@@ -48,14 +44,6 @@ type OptFunc func(*State)
...
 func New(ctx context.Context, l logger.Logger, opts ...OptFunc) (*State, error) {
-	authzQueryParsed, err := ast.ParseBody(authzQuery)
-	if err != nil {
-		return nil, errors.Wrapf(err, "parse query %q", authzQuery)
-	}
-	filteredPairsQueryParsed, err := ast.ParseBody(filteredPairsQuery)
-	if err != nil {
-		return nil, errors.Wrapf(err, "parse query %q", filteredPairsQuery)
-	}
 	authzProjectsV2QueryParsed, err := ast.ParseBody(authzProjectsV2Query)
```
Comment (msorens) [inline components/authz-service/engine/opa/opa.go:47]:
```suggestion
	authzProjectsQueryParsed, err := ast.ParseBody(authzProjectsV2Query)
```

File: components/authz-service/README.md (hunk 1 above)
Comment (tylercloke) [reply components/authz-service/README.md:14]: @bcmdarroch Markdown TOC actually did the above :S Not sure why. I'll drop the table for now.

**Pull request merged**